ecs_deploy/cli.py

```diff
@@ -168,7 +170,7 @@ def cron(cluster, task, rule, image, tag, command, env, role, region, access_key
 
         td.set_images(tag, **{key: value for (key, value) in image})
         td.set_commands(**{key: value for (key, value) in command})
-        td.set_environment(env)
+        td.set_environment(env, env_file)
```

Comment:
Related to my last comment: here you either have to provide either a value for `exclusive_env` or provide the explicit argument name to pass the `env_file`. 

In fact, it would be great to add the `--exclusive-env` option to this operation as well, than the whole `set_environment` call is configurable as in the other operations.